packages/thirdparty-widgets/src/components/ThemeIcon/index.tsx

```diff
@@ -0,0 +1,47 @@
+import cn from "classnames";
+import { useMemo } from "react";
+import { getIconUrl, getSuffix } from "./utils";
+import { Props } from "./types";
+
+const Svg = ({
+  className,
+  href
+}: {
+  className?: string;
+  href: string;
+}): JSX.Element => (
+  <svg className={className}>
+    <use href={href} />
+  </svg>
+);
+
+const ThemeIconPreview = (props: Props): JSX.Element => {
```

Comment:
need to review
1. Documentation
2. How we can use it from the Editor(edit, view mode)